HW0/P6/P6B.py

```diff
@@ -16,16 +17,25 @@ def burnTime(t):
 
     # Use a variety of wait times
     ratio = []
-    wait_time = []
+    # Get 16 random wait times
+    wait_time = sorted([random.uniform(.00000000001, .001) for i in range(16)])
```

Comment:
Your example of translating some text is very vivid. I haven't thought of such suitable example before.